src/adapter.js

```diff
@@ -445,12 +516,17 @@ export function parse({minim, source, generateSourceMap}, done) {
         }
 
         // Currently, default responses are not supported in API Description format
-        // TODO: [Annotation] Add warning about not showing the default!
         const relevantResponses = _.chain(methodValue.responses)
           .omit('default')
           .omit(isExtension)
           .value();
 
+        if (methodValue.responses && methodValue.responses.default) {
```

Comment:
My mistake! Thought lodash was already there. Please ignore me in these situations :)